client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -159,10 +166,11 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 								}
 							)
 						}</span>
+						{ ! ciraAgreementAccepted ? <FormInputValidation text={ translate( 'Required' ) } isError={ true } /> : null }
```

Comment:
This was indeed clever. With the checkbox defaulting off again -- this becomes a little weird though -- an immediate "error" on form load because it's `false`.  Can we just remove now and rely on the disable submit or is that too subtle?